### PR TITLE
#159396462 Add badges and Travis integration

### PR DESCRIPTION
**Pull request by ja-odur**
#### What does this PR do?
* Adds Travis integration and badges to the develop branch.

#### Description of Task to be completed?
* Add Travis Integration
* Add Build, Test Coverage, Maintainability and License badges.
* Add Project name in the `README.rst`.
#### How should this be manually tested?
* On [project GitHub repo](https://github.com/andela/wg-wits), check out branch [ch-add-badges-159396462](https://github.com/andela/wg-wits/tree/ch-add-badges-159396462).
* Scroll down to the README.srt section inorder to see the badges. *Note* Some of the badges have the status `Unknown` because they target `develop` branch.
#### What are the relevant pivotal tracker stories?
[159396462](https://www.pivotaltracker.com/story/show/159396462)
#### Screenshot
screen shot 2018-08-02 at 14 12 38

### CODE REVIEW

**Comment by solnsubuga** [comment]
@ja-odur the badges are not being displayed, they are just links. See attached image. Check on this.
screen shot 2018-07-31 at 09 51 02

File: wger/settings_global.py [hunk 1]
```diff
@@ -90,17 +90,30 @@
...
-    'metrics-graphics',
     'devbridge-autocomplete#1.2.x',
+    'jquery#2.1.x',
+    'metrics-graphics',
+    'shariff#1.24.1',
     'sortablejs#1.4.x',
+    'tinymce',
+    'tinymce-dist',
+
+#    'jquery#2.1.x',
```
Comment (solnsubuga) [inline wger/settings_global.py]: @ja-odur please remove these commented out lines (106 -116) since you already have them covered above.

**Review by solnsubuga — CHANGES_REQUESTED** [review]

**Review by solnsubuga — APPROVED** [review]

**Comment by lym** [comment]
@ja-odur Add a _how can this be manually tested_ section and screenshots of how the badges are being rendered in the README in your PR description.

**Comment by lym** [comment]
@ja-odur Please add the project name to the README, just before the badges, as is done in the picture below.

image

**Review by wasswa-derick — COMMENTED** [review]
@ja-odur 
I have reviewed the PR. It seems to be a great output save for the concern I have about the badges.

In the ReadMe, they are included as images
-  image:: https://travis-ci.org/andela/wg-wits.svg?branch=develop
    :target: https://travis-ci.org/andela/wg-wits

I have made some reading that github could sometimes cache those image badges and even when the metrics change, the badge remains the same. https://github.com/github/markup/issues/224

I would recommend including the Markdown badge that may not be cached for the most part. You can get the markdown by clicking on the badges and look for the representation of that badge in the markdown format.

**Comment by ja-odur** [comment]
@wasswa-derick,
The README is in the .rst format which means the markdown format cannot be rendered in it.
I am still researching on how to prevent the cache issue but those badges are the generated RST formats from both coveralls and codeclimate.

**Comment by RrNn** [comment]
The badges look good to me now. Me I approve merging

**Comment by lym** [comment]
@ja-odur Update the screenshot and description to exclude information about Gitter.

**Pull request merged**